rails/templates/default/database.yml.erb

```diff
@@ -13,5 +13,8 @@
 <% if @database[:port] -%>
   port: <%= @database[:port].to_i.inspect %>
 <% end -%>
+<% if @database[:schema_search_path] -%>
+  schema_search_path: <%= @database[:schema_search_path].to_s.inspect %>
```

Comment:
The .to_s handles the nil case by converting the string to an empty string.  The .inspect handles quoting, so confusing characters don't end up in the yml.  Likely unnecessary 99% of the time, but that 1% means :boom: 

I _think_ a simpler equivalent is `"#{@database[:schema_search_path]}"` but perhaps the thinking is that this _always_ allocates a new string, whereas .to_s only allocates a new string if the source type isn't a string.  So `.to_s.inspect` is likely more efficient by some metric that doesn't matter.